### ISSUE TITLE
Question about cql baseline

### ISSUE THREAD

**Issue by Kuangqi927** [JasonMa2016/CODAC#2]
Hi, 
I have read your excellent work and try to reproduce it. But，I was little confused about the results of cql baseline.

In **hopper-medium-v0** and **hopper-medium-replay-v0**(mixed), I got the normalized socre: **98**(above CODAC level) and **97**(at CODAC level), but the cql baseline in the paper are **58** and **48.6**.
In  **halfcheetah-medium-expert-v0**, I got the normalized socre: **39**, but  **62.4** in the paper.

Besides, I also noticed that the cql results run by your code are different from CQL(Aviral Kumar,2020) paper.

How can I solve this problem? Hope for your reply!

**Comment by JasonMa2016** [comment]
Hi! 

Thank you for your interest in our work! The CQL results we report in our paper are directly taken from (Kumar et al, 2020). We did not run our implementation of CQL on the D4RL benchmarks, so it is expected and quite possible that the results you obtain are different from the reported CQL results. Finally, we note that the CQL implementation in our codebase is for reference and debugging purpose only, since we noticed that the original CQL implementation does not run properly on pytorch 1.6 (see [this issue](https://github.com/aviralkumar2907/CQL/issues/5)). Therefore, due to a slightly different order of backpropagations we implement to be compatible with newer versions of pytorch, results may differ qualitatively compared to the original implementation.
